sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suites.scala

```diff
@@ -874,6 +874,22 @@ class HiveThriftBinaryServerSuite extends HiveThriftJdbcTest {
       assert(rs.getString(1) === expected.toString)
     }
   }
+
+  test("SPARK-26533: Support query auto timeout cancel on thriftserver") {
```

Comment:
Could you check if it can throw an exception (when hive1.2 used internally) by using `HiveUtils.builtinHiveVersion` or the other related variables?

https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/125039/testReport/org.apache.spark.sql.hive.thriftserver/HiveThriftBinaryServerSuite/SPARK_26533__Support_query_auto_timeout_cancel_on_thriftserver/
```
Error Message
java.sql.SQLException: Method not supported
Stacktrace
sbt.ForkMain$ForkError: java.sql.SQLException: Method not supported
	at org.apache.hive.jdbc.HiveStatement.setQueryTimeout(HiveStatement.java:739)
	at org.apache.spark.sql.hive.thriftserver.HiveThriftBinaryServerSuite.$anonfun$new$97(HiveThriftServer2Suites.scala:880)
	at 
```

Reply:
fixed, thanks